integration/js/utils/CloudWatch.js

```diff
@@ -4,13 +4,13 @@ var cloudWatch = new AWS.CloudWatch({
   apiVersion: '2010-08-01'
 });
 
-module.exports.emitMetric = async (namespace, capabilities, metric_name, value, namespaceInfix) => {
+module.exports.emitMetric = async (namespace, capabilities, metric_name, value, cwNameSpaceInfix) => {
```

Comment:
It's kinda confusing to me that one argument is named as `namespace` and the other is named as `cwNameSpaceInfix`.

Because it's feel like there are two different namespaces. One is `namespace`, the other is `cwNamespace`.
But what `cwNameSpaceInfix` does is to add an string segment into `namespace`.

Reply:
I agree with this, we should probably also rename `namespace` to `cwNamespace`